TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/test/java/org/opensearch/migrations/transform/Es2OsPost23Test.java

```diff
@@ -0,0 +1,5 @@
+package org.opensearch.migrations.transform;
+
+public class Es2OsPost23Test {
+
+}
```

Comment:
It looks like the excision test is still in place & I presume that it is passing.  Could you either delete this file or move the excision test into here?

Also please write a test that shows that your new transform fixes the bug that was reported.  You can take a look at the `normalizeJson` functions in [this PR](https://github.com/opensearch-project/opensearch-migrations/pull/433/files) and do that here with hardcoded input/output.  (IntelliJ will escape as necessary if you paste INTO a string)